modules/network-observability-flowcollector-about.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+
+// * configuring-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-flowcollector-about{context}"]
+
+= About the FlowCollector resource 
+The  `FlowCollector` resource is deployed at the time of installation. You can edit yaml directly in the {product-title} web console. In the web console, navigate to *Operators* -> *Installed Operators*. Then under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. Next, select *cluster* then select the *YAML* tab. There, you can modify the `FlowCollector` resource to configure the Network Observability operator. Some of the notable configurations are documented in this documentation. 
```

Comment:
```suggestion
The FlowCollector` resource is deployed at the time of installation. You can edit yaml directly in the {product-title} web console. In the web console, navigate to *Operators* -> *Installed Operators*. Then under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. Next, select *cluster* then select the *YAML* tab. There, you can modify the `FlowCollector` resource to configure the Network Observability operator. Some of the notable configurations are documented in this documentation. 
```

Reply:
@nathan-weinberg i don't get this suggestion... mistake?

Reply:
whoops, yes @jotak - fixed now :)